include/Color.hpp

```diff
@@ -65,6 +66,16 @@ class Color : public ::Color {
         return ::ColorToInt(*this);
     }
 
+    inline std::string ToString() const {
+        std::ostringstream oss;
+        oss << "Color(" << r << ", " << g << ", " << b << ", " << a << ")";
+        return oss.str();
```

Comment:
This is great! We could likely save having to bring in `ostringstream` by leveraging raylib's `TextFormat()`.

Reply:
Good point!